pkg/kube/snapshot/types.go

```diff
@@ -0,0 +1,35 @@
+package snapshot
+
+// SnapshotMeta is metadata of snapshot which is configurable.
+type SnapshotMeta struct {
+	// Name must be unique within a namespace. Is required when creating resources, although
+	// some resources may allow a client to request the generation of an appropriate name
+	// automatically. Name is primarily intended for creation idempotence and configuration
+	// definition.
+	// Cannot be updated.
+	// More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names#names
+	// +optional
+	Name string `json:"name,omitempty" protobuf:"bytes,1,opt,name=name"`
```

Comment:
@saima-s looks like you have copies the comments from `meta/v1.ObjectMeta` struct. We don't have to elaborate this much here. Let's add simple and short comments on fields of the struct. Also, no need to protobuf tags, they are irrelevant in our case.

Reply:
ok, sure.